nu0Box.txt

```diff
@@ -0,0 +1,85 @@
+# betaPara
```

Comment:
NU0BOX.DAT would be a better file name, we use the .dat suffix for other similar files, and have all-caps for the ionosphere parameter .DAT files.